wiki/markdown/configuration.md

```diff
@@ -45,5 +49,111 @@ URI schemes that are accepted are 'file','http' and 'https'.
+
+Consider the following query:
+
+    {
+      "criteria": {
+        "object.identity":"pkg:maven/com.mycompany.myproduct/sub-system@1.1.0"
+      }
+    }
+
+Eiffel Intelligence will replace the prefix "object." with the name of the aggregated
+object (in this case "aggregatedObject"), which is defined by **aggregated.object.name**.
+Read more examples about [queries here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query.md).
+
+
+### Testing aggregation rules
+
+To test new rules for Eiffel Intelligence the property **testaggregated.enabled**
+can be set to true. This gives users the possibility to try out different rule sets
+on a specific set of Eiffel events and see the resulting aggregated object.
+
+
+### Wait list
+
+If Eiffel Intelligence receives events that are not connected to any
+existing aggregated objects, and it is not declared as start event in the
+rules, the event is stored in the wait list database for a while. The
+"time-to-live" value limits how long an event is stored in the wait list.
+
+* waitlist.collection.name
+* waitlist.collection.ttlValue (*seconds*)
+
+It is possible to configure how often Eiffel Intelligence should handle
+unprocessed events lying in the wait list. These properties are in **milliseconds**:
+
+* waitlist.initialDelayResend (*milliseconds*)
+* waitlist.fixedRateResend (*milliseconds*)
+
+
+## Subscriptions
+
+Eiffel Intelligence stores subscriptions in a database with the collection
+name configured with the property **subscription.collection.name**. When
+subscriptions have fulfilled their requirements and notifications
+should not be repeated, the subscription is stored together with the
+matched aggregated object in the database. The name of this particular
+collection is defined by the below property:
+
+* subscription.collection.name
+* subscription.collection.repeatFlagHandlerName
+
+
+### Notifications
+
+**email.sender** defines who should be the sender of the email Eiffel
+Intelligence sends when a subscription is triggered.
+The value for email.subject is used by all subscriptions which have been
+created via the REST API. When a subscription is created using the Eiffel
+Intelligence front-end GUI it is possible to set individual email subject
+for each subscription.
+
+* email.sender
+* email.subject
+
+
+### Missed notifications
+
+Should the subscription notification for some reason fail, the notification
+is stored in a database, and **missedNotificationDatabaseName** property
+defines what that database should be called. It is also possible to configure
+what the collection name for missed notifications should be. **notification.ttl.value** tells
+Eiffel Intelligence how long a missed notification will be stored in the
+database before deletion. With **notification.failAttempt** property, it
+is possible to configure the number of attempts Eiffel Intelligence will
+retry to make REST POST notification when a subscription is filled.
```

Comment:
Instead of "filled" -> "created or edited." is maybe more clear for the user.
Or do you mean something specific with "filled" word in the subscription?

Reply:
triggered is the word you are looking for, a Subscription gets triggered and then it sends it calls.

Reply:
Noted, will fix.

Reply:
Noted, will fix